Automattic/wp-calypso
# composite-checkout: Use tax location information for shopping cart

#### Changes proposed in this Pull Request

Tax location information _is_ currently sent from the form fields to the transaction endpoint, causing the price to reflect taxes if they are required for that location. However, that same data is never sent to the shopping cart endpoint, which means that the price displayed never includes taxes.

This PR adds support for sending the tax location data from the contact form to the shopping cart endpoint when it changes, and then using that data for the taxes line item. It also respects the `display_taxes` property sent back by the server which serves to differentiate between a tax rate of 0 and no taxes being applied; the tax line item won't display at all if that property is false (mirroring the behavior of current checkout).

#### Testing instructions

- Add a plan to your cart.
- Enter a taxable location in the contact form (eg: postal code 10001, United States).
- Visit the review step and verify that taxes are displayed.

![Screen Shot 2020-02-12 at 12 30 21 PM](https://user-images.githubusercontent.com/2036909/74360654-782d7e00-4d93-11ea-8c94-b3f12d6c7c72.png)
